xmille/comp.c

```diff
@@ -62,9 +62,9 @@ calcmove(void)
 	foundend = FALSE;
 
 	/* Try for a Coup Forre, and see what we have. */
-	for (i = 0; i < NUM_CARDS; i++)
+	for (int i = 0; i < NUM_CARDS; i++)
```

Comment:
should probably just declare 'int i' at function level instead of relying on C99?